src/storage.js

```diff
@@ -0,0 +1,192 @@
+import { _, console } from './utils'
+
+var DOMAIN_MATCH_REGEX = /[a-z0-9][a-z0-9-]+\.[a-z.]{2,6}$/i
+
+// Methods partially borrowed from quirksmode.org/js/cookies.html
+export const cookieStore = {
+    get: function (name) {
+        try {
+            var nameEQ = name + '='
+            var ca = document.cookie.split(';')
+            for (var i = 0; i < ca.length; i++) {
+                var c = ca[i]
+                while (c.charAt(0) == ' ') {
+                    c = c.substring(1, c.length)
+                }
+                if (c.indexOf(nameEQ) === 0) {
+                    return decodeURIComponent(c.substring(nameEQ.length, c.length))
+                }
+            }
+        } catch (err) {}
+        return null
+    },
+
+    parse: function (name) {
+        var cookie
+        try {
+            cookie = _.JSONDecode(cookieStore.get(name)) || {}
+        } catch (err) {
+            // noop
+        }
+        return cookie
+    },
+
+    set_seconds: function (name, value, seconds, cross_subdomain, is_secure) {
+        try {
+            var cdomain = '',
+                expires = '',
+                secure = ''
+
+            if (cross_subdomain) {
+                var matches = document.location.hostname.match(DOMAIN_MATCH_REGEX),
+                    domain = matches ? matches[0] : ''
+
+                cdomain = domain ? '; domain=.' + domain : ''
+            }
+
+            if (seconds) {
+                var date = new Date()
+                date.setTime(date.getTime() + seconds * 1000)
+                expires = '; expires=' + date.toGMTString()
+            }
+
+            if (is_secure) {
+                secure = '; secure'
+            }
+
+            document.cookie = name + '=' + encodeURIComponent(value) + expires + '; path=/' + cdomain + secure
+        } catch (err) {
+            return
+        }
+    },
```

Comment:
I couldn't find anything using this function. Dead code?

Reply:
Seems so, great catch!